frontend/include/chpl/resolution/resolution-queries.h

```diff
@@ -408,7 +408,7 @@ CallResolutionResult resolveCallInMethod(ResolutionContext* rc,
   as the point-of-instantiation scopes that were used when resolving them.
  */
 CallResolutionResult resolveGeneratedCall(Context* context,
-                                          const uast::AstNode* astForErr,
+                                          const uast::AstNode* astForErrAndPoi,
```

Comment:
IMO if it's used for multiple things it's better to have a short name like `astContext` and then describe what it's for in the docs comment.